modules/api-server/src/server/services.ts

```diff
@@ -188,6 +188,18 @@ function fillServiceMetadata(api: any, baseUrl: any) {
     }
     let labels = api.labels ? api.labels : {};
     labels["type"] = api.type == "odata" ? "OData" : "OpenAPI"
+    if (api.type == "odata") {
+        labels["type"] = "OData v" + 2
+    }
+    else if (apiData.spec.openapi) {
+        labels["type"] = "OpenAPI v" + apiData.spec.openapi

```

Comment:
I would really prefer to use only the major version, can you add a simple parsing logic?